packages/translations/src/index.ts

```diff
@@ -22,22 +22,25 @@ import type {
  * @returns boolean which indicates if the languages are loaded
  */
 export function useTranslations(initParams: TranslationInitParams): boolean {
-  const [initialized, setInitialized] = useState(i.isInitialized)
+  const [initialized, setInitialized] = useState(false)
+  const [loaded, setLoaded] = useState(false)
```

Comment:
Is it possible that initialized occurs after loaded? If not, I would only keep state for loaded.

Reply:
I updated it now to something similar, wdyt? 

it now uses the callback instead of the events